### ISSUE TITLE
(Suggestion) Toggle visibility of second hotbar

### ISSUE THREAD

**Issue by ramonpdx03** [DenWav/ExtendedHotbar#11]
I think it would be nice if it was possible to have the feature of swapping hotbars with just the main one on the screen

An option on the mod menu that is disabled by default

**Comment by DenWav** [comment]
Are you saying to swap items even when the top hotbar isn't visible?

**Comment by ramonpdx03** [comment]
> Are you saying to swap items even when the top hotbar isn't visible?

Yes! I think it would be nice to have this feature with the standard hotbar layout
